pattern-1/bosh-release/README.md

```diff
@@ -1,242 +1,96 @@
-
-        echo -e "\e[32m>> Setting alias for the environment... \e[0m"
-        bosh -e 192.168.50.6 alias-env vbox --ca-cert <(bosh int vbox/creds.yml --path /director_ssl/ca)
-
-9. Login to BOSH Director using generated password
-
-        echo -e "\e[32m>> Loging in... \e[0m"
-        bosh -e vbox login --client=admin --client-secret=$(bosh int vbox/creds.yml --path /admin_password
-
-10. Upload binaries as blobs.
+2. Navigate to `pivotal-cf-apim/pattern-1/bosh-release` directory.
 
-        cd pivotal-cf-apim/bosh-release/ 
-        echo -e "\e[32m>> Adding blobs... \e[0m"
-        bosh -e vbox add-blob ../../jdk-8u144-linux-x64.tar.gz oraclejdk/jdk-8u144-linux-x64.tar.gz
-        bosh -e vbox add-blob ../../mysql-connector-java-5.1.24-bin.jar mysqldriver/mysql-connector-java-5.1.24-bin.jar
-        bosh -e vbox add-blob ../../wso2am-2.1.0.zip wso2apim/wso2am-2.1.0.zip
-        bosh -e vbox add-blob ../../wso2am-analytics-2.1.0.zip wso2apim_analytics/wso2am-analytics-2.1.0.zip
+3. Add the following software distributions to the `dist` folder.
 
-        echo -e "\e[32m>> Uploading blobs... \e[0m"
-        bosh -e vbox -n upload-blobs
+- [JDK 1.8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 
-11. Create APIM BOSH release and upload it to BOSH Director.
+- [MySQL JDBC driver](https://dev.mysql.com/downloads/connector/j/5.1.html)
 
-        echo -e "\e[32m>> Creating bosh release... \e[0m"
-        bosh -e vbox create-release --force
+- WSO2 API Manager WUM updated product distribution
 
-        echo -e "\e[32m>> Uploading bosh release... \e[0m"
-        bosh -e vbox upload-release
+- WSO2 API Manager Analytics WUM updated product distribution
 
-12. Download latest bosh-lite warden stemcell from bosh.io and upload it to BOSH Director.
-
-        if [ ! -f bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz ]; then
-            echo -e "\e[32m>> Stemcell does not exist! Downloading... \e[0m"
-            wget https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-        fi
-
-        echo -e "\e[32m>> Uploading Stemcell... \e[0m"
-        bosh -e vbox upload-stemcell bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-
-13. Deploy the WSO2 API Manager BOSH release manifest in BOSH Director.
-
-        echo -e "\e[32m>> Deploying bosh release... \e[0m"
-        yes | bosh -e vbox -d wso2apim deploy wso2apim-manifest.yml
-
-14. Add route to VirtualBox network.
-
-        echo -e "\e[32m>> Adding route... \e[0m"
-        sudo route add -net 10.244.0.0/16 gw 192.168.50.6
-
-15. List VMs.
-
-        echo -e "\e[32m>> Listing VMs... \e[0m"
-        bosh -e vbox vms
-
-
-[1]: http://bosh.io
-[2]: https://github.com/bhathiya/pivotal-cf-apim
-[3]: https://github.com/cloudfoundry/bosh-deployment 
+4. Execute the deploy.sh script.
+   ```
+   ./deploy.sh
```

Comment:
IMO it would be better to remove the deploy.sh helper script and provide the required commands in the README.md file.

Reply:
@imesh the helper scripts would be removed in the future changes. And as planned the guidelines would be moved to the README.md file.